player/tests/test.rs

```diff
@@ -128,7 +129,7 @@ impl Test<'_> {
             println!("\t\t\tChecking {}", expect.name);
             let buffer = wgc::id::TypedId::zip(expect.buffer.index, expect.buffer.epoch, backend);
             let (ptr, size) =
-                wgc::gfx_select!(device => global.buffer_get_mapped_range(buffer, expect.offset, None))
+                wgc::gfx_select!(device => global.buffer_get_mapped_range(buffer, expect.offset, Some(BufferSize::new(expect.data.len() as u64).unwrap())))
```

Comment:
We should probably just do `BufferSize::new()` without `unwrap`/`Some` here. There is only so much we can do against a bug in the test itself, no need to be over-protective in this piece of code.